react-native-reanimated.d.ts

```diff
@@ -458,6 +458,10 @@ declare module 'react-native-reanimated' {
       colorSpace?: 'RGB' | 'HSV'
     ): string | number;
 
+    export function makeMutable<T>(
+      initialValue: T
+    ): SharedValue<T>;
```

Comment:
shouldn't be `T extends SharedValueType`?

Reply:
Indeed, I fixed it.